frontend/shared/src/components/forms/heading/Heading.tsx

```diff
@@ -1,27 +1,16 @@
 import { LoadingSpinner, Tooltip } from 'hds-react';
 import * as React from 'react';
-import { DefaultTheme } from 'styled-components';
 
-import { $Header } from './Heading.sc';
-
-type HeadingProps = {
-  size?: keyof DefaultTheme['fontSize']['heading'];
-  as?: 'h1' | 'h2' | 'h3';
-  header?: string;
-  loading?: boolean;
-  loadingText?: string;
-  loadingFinishedText?: string;
-  tooltip?: string;
-};
+import { $Header, HeadingProps } from './Heading.sc';
 
 const Heading: React.FC<HeadingProps> = ({
   as,
-  size,
+  size = 'l',
```

Comment:
Changing the default size from `'m'` to `'l'` may affect other components, have you checked that?

Reply:
I tried to remove it and couldn't see difference in my app. So I think it safer to remove, so no accidental changs to benefit.